### DISCUSSION TITLE
statsd_datadog_enabled in [metrics]

### DISCUSSION THREAD

**Discussion by nicolamarangoni** [apache/airflow#26940] [Q&A]
Hello, the docs by DataDog seems a bit obsolete: https://docs.datadoghq.com/integrations/airflow/?tab=host
There is no mention of `statsd_datadog_enabled` in `[metrics]`.
I'm using the latest version of the datadog-agent running in a sidecar container in my pod. In the DD-agent container, a statsd server called DogStatsD runs per default on `localhost:8125`. neither `statsd_on` nor the settings in `datadog.yaml` as per documentation by DataDog look to be necessary, `statsd_datadog_enabled = True` just do the job.
However I don't find any documentation about this new option. Are there any docs about it?
Here there is no mention of it: https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/check-health.html#check-health-scheduler-health-check-server

**Comment by potiuk** [comment]
Looked at the code and it looks like the most you have is in the configuration https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#statsd-datadog-enabled . 

But if you experiment and find out more details how it works @nicolamarangoni , I think you are very good candidate to add such documentation. That's a good way to "pay back" for a free software and contribute back. 

You can very easily join > 2200 contributors we have (mostly people like you, who missed something and added it back  - for example docs when they found out how it works). And it's super-easy, just click "Suggest a change on this page" in 
https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/check-health.html#check-health-scheduler-health-check-server and you will open Pull Request and you will be able to add the PR for docs directly from GitHub UI without setting up any environment. Looking forward to it !

**Comment by kzap** [comment]
DataDog has this PR in their documentation recommending to set it to False https://github.com/DataDog/integrations-core/pull/16378

**Comment by potiuk** [comment]
An yes. PR to contribute any improvements for others looking into uit are still definitely welcome. We are nearing 3000 contributors now and contributing nice documentation to make it easier for other users by someone who is a user of the feature itself is a great idea.
